cmd/main.go

```diff
@@ -29,37 +30,41 @@ import (
 	flag "github.com/spf13/pflag"
 )
 
+// A stringSet is a set of strings.
+type stringSet map[string]struct{}
+
 var (
 	orgName        string
 	configFilename string
 	force          bool
 	dryRun         bool
 	addUsers       []string
 	addTeams       []string
+	setBackporter  []string
+	setJanitor     []string
+	setTriage      []string
+	addPTO         []string
+	removePTO      []string
 )
 
 func init() {
 	flag.StringVar(&orgName, "org", "cilium", "GitHub organization name")
-	flag.StringVar(&configFilename, "config-filename", "team-assignments.yaml", "GitHub organization and repository names separated by a slash")
+	flag.StringVar(&configFilename, "config-filename", "team-assignments.yaml", "Config filename")
 	flag.BoolVar(&force, "force", false, "Force local changes into GitHub without asking for configuration")
 	flag.BoolVar(&dryRun, "dry-run", false, "Dry run the steps without performing any write operation to GitHub")
 	flag.StringSliceVar(&addUsers, "add-users", nil, "Adds new users to the configuration file")
 	flag.StringSliceVar(&addTeams, "add-teams", nil, "Adds new teams to the configuration file")
+	flag.StringSliceVar(&setBackporter, "set-backporter", nil, "Sets the the members of the backporter team")
+	flag.StringSliceVar(&setJanitor, "set-janitor", nil, "Sets the the members of the janitor team")
+	flag.StringSliceVar(&setTriage, "set-triage", nil, "Sets the the members of the triage team")
+	flag.StringSliceVar(&addPTO, "add-pto", nil, "Add users on PTO")
+	flag.StringSliceVar(&removePTO, "remove-pto", nil, "Remove users from PTO")
 	flag.Parse()
-
-	go signals()
-}
-
-var globalCtx, cancel = context.WithCancel(context.Background())
-
-func signals() {
-	signalCh := make(chan os.Signal, 1)
-	signal.Notify(signalCh, os.Interrupt)
-	<-signalCh
-	cancel()
 }
 
```

Comment:
Why was this removed? :thinking:

Reply:
It breaks normal SIGINT handling. Normally Ctrl+C sends SIGINT which interrupts the process. With this signal handler Ctrl+C is ignored (it cancels the `globalCtx`, but this only has an effect if `globalCtx` has an operation in process). See the second commit in the PR for a bit more info.

Reply:
Can't we fix the intended behavior instead? Aborting the program with Ctrl+C.

Reply:
The default Linux/Go behavior (i.e. without the bespoke `SIGINT` handling) is already the correct behavior. There is no need to trap `SIGINT`: the operating system will automatically clean up all associated resources.

Put differently: what are the reasons for including the bespoke `SIGINT` handling?

Reply:
@twpayne IIRC `SIGINT` wouldn't interrupt the program without this, i.e., the github requests would still keep going.

Reply:
OK, I've removed this commit.